src/patient/patientSagas.jsx

```diff
@@ -126,12 +126,23 @@ function* initiateGetIC3PatientsAction(action) {
   }
 }
 
+function* sessionInitiateGetIC3PatientsAction(action) {
```

Comment:
@mogoodrich When we change the location in the app, the session resets and the selected patient in the store is cleared, and it affects the checkin location ticket, that's why I had to make this change.

Reply:
@jomadoye  not quite sure we want to remove this... any time we change location, we do want to clear out the existing patients in the store.  I do see how this could be problematic though, especially if we try to change the location when in the context of a specific patients... we may want to disallow this.  Can you explain how I can reproduce any bug you are seeing around this?

Reply:
@mogoodrich For MLW-942. if we select a patient and go to the screening page or summaryPage and then go on to change the location, the app crashes mostly because there is no patient in the store.

Reply:
@jomadoye that makes sense... just talked with Dave about this... let's keep the clearPatientStore() but add a trigger here to this saga to redirect to the home page... that is, whenever the user changes the session location, we redirect to the home page.  Really, the user should be rarely changing the session in real life.

Reply:
Updated @mogoodrich. I'll merge this in.

Reply:
Sounds good, thanks!